### DISCUSSION TITLE
Nuevo en Launch

### DISCUSSION THREAD

**Discussion by frankYanez** [community/community#38675] [Other Feature Feedback, Questions, & Ideas]
Hola buen dia. Vi que eres ino de los administradores en el launch de innovacion mx. No me permite entrar a mi cuenta, envie correos al contacto pero no obtengo respuesta. Podrias ayudarme por favo?

**Comment by ossanna16** [comment]
Hi there @frankYanez and welcome to our community! Can you please elaborate a bit further regarding your issue so we can help?
